environments/production/inventory.ini

```diff
@@ -1,5 +1,5 @@
 [proxy1]
-10.202.20.47 hostname=proxy1-production ufw_private_interface=ens5 ansible_python_interpreter=/usr/bin/python3 public_ip=100.24.216.221
+10.202.20.47 hostname=proxy1-production ufw_private_interface=ens5 ansible_python_interpreter=/usr/bin/python3 public_ip=
```

Comment:
is `public_ip` not needed?

Reply:
I have changed public_ip={{ aws_resources['proxy0-production.public_ip'] }} to public_ip={{ aws_resources['proxy1-production.public_ip'] }} which is updating proxy public ip after running aws-fill-inventory